WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
I never said that. I agree on the arbitrary dates, but what you did alone without concertation isn't a solution. It needs to be discussed thoroughly beforehand.